example/example/autoextract.py

```diff
@@ -21,6 +22,10 @@ class AutoextractProductResponse:
 
 @provides(AutoextractProductResponse)
 class AutoextractProductProvider(PageObjectInputProvider):
+
+    def __init__(self, response: DummyResponse):
```

Comment:
What do you think about making it explicit that this thing needs an URL, not a DummyResponse? Not sure how best to do it.

Reply:
I'm not sure. In any case, DummyResponse instance does have a valid URL and references the original request so that users/developers could consult meta and other attributes.